src/main/java/ua/softserveinc/tc/util/DateUtil.java

```diff
@@ -41,6 +43,10 @@ public static Date toDate(String date) {
         }
     }
 
+    public static Date toDate(LocalDate date) {
```

Comment:
Why are you need start of this date? And why are you call that as toDate(), but not as to StartIfDate() ?